core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/CreatingBillingProjectMonitor.scala

```diff
@@ -90,63 +90,30 @@ trait CreatingBillingProjectMonitor extends LazyLogging {
-                }
-              }
-            } yield {
-              updatedProject
+            gcsDAO.cleanupDMProject(project.projectName) map { _ =>
+              project.copy(status = CreationStatuses.Ready)
             }
 
-          case Seq(RawlsBillingProjectOperationRecord(_, gcsDAO.CREATE_PROJECT_OPERATION, _, true, Some(error), _)) =>
+          case fail@Seq(RawlsBillingProjectOperationRecord(_, gcsDAO.DEPLOYMENT_MANAGER_CREATE_PROJECT, _, true, Some(error), _)) =>
             // create project operation finished with an error
-            logger.debug(s"project ${project.projectName} creation finished with errors: $error")
-            Future.successful(project.copy(status = CreationStatuses.Error, message = Option(error)))
-
-          case operations: Seq[RawlsBillingProjectOperationRecord] if operations.forall(rec => rec.done && rec.errorMessage.isEmpty) =>
-            // all operations completed successfully
-            for {
-              ownerGroupEmail <- UserService.getGoogleProjectOwnerGroupEmail(samDAO, project)
-              computeUserGroupEmail <- UserService.getComputeUserGroupEmail(samDAO, project)
-              updatedProject <- gcsDAO.completeProjectSetup(project, Set(ownerGroupEmail, computeUserGroupEmail)) map {
-                case util.Failure(t) =>
-                  logger.info(s"Failure completing project for $project", t)
-                  project.copy(message = Option(t.getMessage))
-                case Success(_) => project.copy(status = CreationStatuses.Ready)
-              }
-            } yield {
-              updatedProject
+            logger.debug(s"project ${project.projectName.value} creation finished with errors: $error")
+            gcsDAO.cleanupDMProject(project.projectName) map { _ =>
+              project.copy(status = CreationStatuses.Error, message = Option(s"project ${project.projectName.value} creation finished with errors: $error"))
             }
 
-          case operations: Seq[RawlsBillingProjectOperationRecord] if operations.forall(rec => rec.done) =>
-            // all operations completed but some failed
-            val messages = operations.collect {
-              case RawlsBillingProjectOperationRecord(_, operationName, _, true, Some(error), _) => s"Failure enabling api $operationName: ${error}"
-            }
-            val errorMessage = messages.mkString("[", "], [", "]")
-            logger.debug(s"errors enabling apis for project ${project.projectName}: $errorMessage")
-            Future.successful(project.copy(status = CreationStatuses.Error, message = Option(errorMessage)))
+          case Seq(RawlsBillingProjectOperationRecord(_, gcsDAO.CREATE_PROJECT_OPERATION, _, _, _, _)) =>
+            Future.successful(project.copy(status = CreationStatuses.Error, message = Option(s"failure processing new project ${project.projectName.value}: old-style project creation no longer supported")))
```

Comment:
what happens to any project creations in flight? Do we just ensure that there aren't any?

Reply:
Yes.